src/components/rating.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../../css/style.css" />
+    <title>Cache UI</title>
+  </head>
+  <body>
+    <div class="ui-sideNav gutter-medium">
+      <div class="ui-sideNav-header text-medium text-color-white">
+        <span class="ui-sideNav-logo gutter-small">C</span>
+        <span class="gutter-medium"
+          ><a
+            href="../../index.html"
+            class="text-color-white text-decoration-none"
+            >UI Docs</a
+          ></span
+        >
+        <i
+          class="fas fa-times-circle ui-sideNav-icon text-medium"
+          aria-hidden="true"
+          onClick="closeSideNav()"
+        ></i>
```

Comment:
this onclick can be all lowercase, it works this way too but better to have in lowercase to not to confuse with react onClick function. same thing in rest of the files.